react/features/connection-indicator/components/web/ConnectionIndicator.js

```diff
@@ -153,59 +124,16 @@ class ConnectionIndicator extends Component<Props, State> {
         super(props);
 
         this.state = {
-            autoHideTimeout: null,
+            autoHideTimeout: undefined,
             showIndicator: false,
             showMoreStats: false,
             stats: {}
         };
 
         // Bind event handlers so they are only bound once for every instance.
-        this._onStatsUpdated = this._onStatsUpdated.bind(this);
         this._onToggleShowMore = this._onToggleShowMore.bind(this);
     }
 
-    /**
-     * Starts listening for stat updates.
-     *
-     * @inheritdoc
-     * returns {void}
-     */
-    componentDidMount() {
-        statsEmitter.subscribeToClientStats(
-            this.props.userID, this._onStatsUpdated);
-    }
-
-    /**
-     * Updates which user's stats are being listened to.
-     *
-     * @inheritdoc
-     * returns {void}
-     */
-    componentDidUpdate(prevProps) {
-        if (prevProps.userID !== this.props.userID) {
-            statsEmitter.unsubscribeToClientStats(
-                prevProps.userID, this._onStatsUpdated);
-            statsEmitter.subscribeToClientStats(
-                this.props.userID, this._onStatsUpdated);
-        }
-    }
-
-    /**
-     * Cleans up any queued processes, which includes listening for new stats
-     * and clearing any timeout to hide the indicator.
-     *
-     * @private
-     * @returns {void}
-     */
-    componentWillUnmount() {
-        statsEmitter.unsubscribeToClientStats(
-            this.props.userID, this._onStatsUpdated);
-
-        if (this.state.autoHideTimeout) {
```

Comment:
@virtuacoplenny, you don't need to do truthy checks for clearTimeout and clearInterval.

Reply:
Truth